controls/srg_ctr/SRG-APP-000386-CTR-000920.yml

```diff
@@ -6,3 +6,13 @@ controls:
     (whitelist) policy to allow only authorized container images in the container
     platform.
   status: inherently met
+  artifact_description: |-
+    Supporting evidence is in the following documentation
+
+    https://docs.openshift.com/container-platform/latest/authentication/using-rbac.html
+  status_justification: |-
```

Comment:
I didn't see this in the draft. Does it need to be there?

Reply:
Umm but it is? Not sure if [this link](https://docs.google.com/spreadsheets/d/1ouRalvXzRHuhPPN_AKMvqqscUqGvVOkkEdMP3YYBO44/edit#gid=1236833936) would go straight to the cell, if not, row 149, column P

Reply:
Aha - found it. Thanks